astropy/coordinates/tests/accuracy/generate_ref_ast.py

```diff
@@ -125,12 +148,15 @@ def ref_fk4_no_e_fk5(fnout="fk4_no_e_fk5.csv"):
     t.add_column(Column(name="dec_fk5", data=dec_fk5))
     t.add_column(Column(name="ra_fk4", data=ra_fk4))
     t.add_column(Column(name="dec_fk4", data=dec_fk4))
-    f = open(os.path.join("data", fnout), "wb")
-    f.write(
-        f"# This file was generated with the {os.path.basename(__file__)} script, and"
-        " the reference values were computed using AST\n"
-    )
-    t.write(f, format="ascii", delimiter=",")
+    formats = {
+        k: "%0.10f" for k in "ra_in,dec_in,ra_fk5,dec_fk5,ra_fk4,dec_fk4".split(",")
```

Comment:
```suggestion
        k: "%0.10f" for k in "ra_in,dec_in,ra_fk5,dec_fk5,ra_fk4,dec_fk4".split(",")
```
```suggestion
        k: ".12g" for k in "ra_in,dec_in,ra_fk5,dec_fk5,ra_fk4,dec_fk4".split(",")
```

Reply:
that's it ! thanks a *LOT*. There's no difference after that, so I'll close this PR and include your suggestion in #16976